integration/src/test/resources/synchronized-flowref-mule-context-start-config.xml

```diff
@@ -16,7 +16,7 @@
     </flow>
 
     <flow name="flow2">
-        <custom-source class="org.mule.test.routing.SynchronizedFlowRefOnMuleContextStartTestCase$UnblockProcessingSource"/>
+        <test:custom-source class="org.mule.test.routing.SynchronizedFlowRefOnMuleContextStartTestCase$UnblockProcessingSource"/>
```

Comment:
can you put the logic from this source into `SkeletonSource` to avoid having this class?